tests/subsys/settings/nvs/raw/prj_qemu_x86.conf

```diff
@@ -0,0 +1,11 @@
+CONFIG_ZTEST=y
```

Comment:
how is this difference from prj_qemu.conf?

Reply:
there is no difference, I will remove it in the next patch